app/Helpers/render/user.php

```diff
@@ -49,8 +37,6 @@ function () use ($username) {
         return "<span class='inline whitespace-nowrap'><span class='inline-block'>" . $iconLabel . $usernameLabel . "</span></span>";
     }
 
-    $username = $user['User'] ?? null;
```

Comment:
Without this, `[user=X]` links are no longer case corrected. `[user=Jamiras]` goes to `user/Jamiras`, but `[user=jamiras]` goes to `user/jamiras`, which does do the case correction at that point, but the link is incorrectly cased.

Reply:
You're right, I didn't recognize that was the intent of this line. Given that, an existence check unfortunately won't be sufficient.

In https://github.com/RetroAchievements/RAWeb/pull/1674/commits/5f637768bc60a06f00d234f902b6c804b12031f7, I've instead opted to utilization the same caching implementation that user cards are using (with a subsequent `toArray()` call within the caching function). Now, at the very least, there will not be caching conflicts.